<event logger="KPIWeb.LogWriter" timestamp="2015-03-25T10:00:05.1688152+03:00" level="ERROR" thread="7" domain="/LM/W3SVC/2/ROOT-1-130717403645804937" username="Кирюха-ПК\Кирюха"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\Кирюха\Desktop\Репа\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-30T09:00:06.4163951+03:00" level="ERROR" thread="9" domain="/LM/W3SVC/2/ROOT-1-130721687697862999" username="Кирюха-ПК\Кирюха"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\Кирюха\Desktop\Репа\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-30T10:55:29.1393522+03:00" level="ERROR" thread="61" domain="/LM/W3SVC/2/ROOT-1-130721687697862999" username="Кирюха-ПК\Кирюха"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\Кирюха\Desktop\Репа\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-30T20:58:54.9107617+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/8/ROOT-1-130722118926784700" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-30T21:00:34.5645892+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/8/ROOT-1-130722120191205621" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-30T21:06:33.9618536+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/8/ROOT-1-130722123683310086" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-30T21:11:23.1597143+03:00" level="ERROR" thread="7" domain="/LM/W3SVC/8/ROOT-1-130722126552882791" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-31T09:04:11.4517223+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/8/ROOT-1-130722554277513667" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-31T09:22:20.8650332+03:00" level="ERROR" thread="7" domain="/LM/W3SVC/8/ROOT-1-130722565226919937" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-31T09:24:53.4987633+03:00" level="ERROR" thread="13" domain="/LM/W3SVC/8/ROOT-1-130722566778888705" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-31T09:37:06.2026716+03:00" level="ERROR" thread="6" domain="/LM/W3SVC/8/ROOT-1-130722573951448952" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-31T09:41:28.4646721+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/8/ROOT-1-130722576134203798" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-31T10:01:56.8959344+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/8/ROOT-1-130722588943456446" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-31T10:28:08.202808+03:00" level="ERROR" thread="13" domain="/LM/W3SVC/8/ROOT-1-130722604658645303" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-31T10:33:16.8674626+03:00" level="ERROR" thread="6" domain="/LM/W3SVC/8/ROOT-1-130722607808375457" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-31T10:41:44.1454772+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/8/ROOT-1-130722612811941645" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-31T10:50:25.5052973+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/8/ROOT-1-130722618096083881" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-31T13:49:45.6568442+03:00" level="ERROR" thread="21" domain="/LM/W3SVC/8/ROOT-1-130722724336361491" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-31T14:01:55.7326021+03:00" level="ERROR" thread="39" domain="/LM/W3SVC/8/ROOT-1-130722724336361491" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-31T14:04:23.6280612+03:00" level="ERROR" thread="38" domain="/LM/W3SVC/8/ROOT-1-130722724336361491" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-31T14:05:42.1475523+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/8/ROOT-1-130722735246325505" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-31T14:06:35.2965923+03:00" level="ERROR" thread="17" domain="/LM/W3SVC/8/ROOT-1-130722735246325505" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-31T14:21:51.7870125+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/8/ROOT-1-130722744648333269" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-31T14:25:08.3142532+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/8/ROOT-1-130722746847989082" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-31T14:32:45.9194267+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/8/ROOT-1-130722751525046595" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-31T14:47:29.208948+03:00" level="ERROR" thread="13" domain="/LM/W3SVC/8/ROOT-1-130722760354491610" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-31T14:54:36.0073595+03:00" level="ERROR" thread="14" domain="/LM/W3SVC/8/ROOT-1-130722764237783722" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-31T15:00:28.6905318+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/8/ROOT-1-130722768082423622" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-31T15:25:00.9497402+03:00" level="ERROR" thread="7" domain="/LM/W3SVC/8/ROOT-1-130722782841747807" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-31T15:25:03.4328822+03:00" level="ERROR" thread="6" domain="/LM/W3SVC/8/ROOT-1-130722782841747807" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-31T15:26:07.7015582+03:00" level="ERROR" thread="19" domain="/LM/W3SVC/8/ROOT-1-130722782841747807" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-31T15:46:50.479641+03:00" level="ERROR" thread="19" domain="/LM/W3SVC/8/ROOT-1-130722795327761967" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-31T16:05:50.998875+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/8/ROOT-1-130722806766136204" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-31T16:30:55.3309179+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/8/ROOT-1-130722821962315376" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-31T16:37:19.4908906+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/8/ROOT-1-130722825599023384" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-03-31T16:37:50.7366777+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/8/ROOT-1-130722825599023384" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-01T11:45:04.3019992+03:00" level="ERROR" thread="7" domain="/LM/W3SVC/2/ROOT-1-130723514722691670" username="Кирюха-ПК\Кирюха"><message>System.NullReferenceException: Ссылка на объект не указывает на экземпляр объекта.    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\Кирюха\Desktop\Репазиторий\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 44KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-01T11:45:29.7594552+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/2/ROOT-1-130723514722691670" username="Кирюха-ПК\Кирюха"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\Кирюха\Desktop\Репазиторий\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-01T14:18:41.992411+03:00" level="ERROR" thread="9" domain="/LM/W3SVC/2/ROOT-1-130723606645971282" username="Кирюха-ПК\Кирюха"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\Кирюха\Desktop\Репазиторий\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-01T16:12:17.4953987+03:00" level="ERROR" thread="9" domain="/LM/W3SVC/2/ROOT-1-130723674976361189" username="Кирюха-ПК\Кирюха"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\Кирюха\Desktop\Репазиторий\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-02T08:51:55.4986397+03:00" level="ERROR" thread="9" domain="/LM/W3SVC/2/ROOT-1-130724274793215705" username="Кирюха-ПК\Кирюха"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\Кирюха\Desktop\Репазиторий\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-02T09:23:48.1670381+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130724292043182347" username="Кирюха-ПК\Кирюха"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\Кирюха\Desktop\Репазиторий\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-02T09:25:44.7357055+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130724292043182347" username="Кирюха-ПК\Кирюха"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\Кирюха\Desktop\Репазиторий\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-03T09:42:34.6738199+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/2/ROOT-1-130725169295953855" username="Кирюха-ПК\Кирюха"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\Кирюха\Desktop\Репазиторий\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-03T15:45:16.917898+03:00" level="ERROR" thread="7" domain="/LM/W3SVC/8/ROOT-1-130725386895893349" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-03T15:53:54.5060158+03:00" level="ERROR" thread="6" domain="/LM/W3SVC/8/ROOT-1-130725392190411313" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-03T16:10:33.0334374+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/8/ROOT-1-130725401984854613" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-03T16:10:54.8696863+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/8/ROOT-1-130725401984854613" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-03T16:11:42.7264236+03:00" level="ERROR" thread="5" domain="/LM/W3SVC/8/ROOT-1-130725401984854613" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-03T16:35:00.3989616+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/8/ROOT-1-130725416732584093" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-03T16:39:05.2613155+03:00" level="ERROR" thread="13" domain="/LM/W3SVC/8/ROOT-1-130725419220649888" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T08:49:43.7168908+03:00" level="ERROR" thread="13" domain="/LM/W3SVC/8/ROOT-1-130727728888634655" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T09:10:50.0203387+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/8/ROOT-1-130727742340384246" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T10:03:08.6898603+03:00" level="ERROR" thread="9" domain="/LM/W3SVC/8/ROOT-1-130727773540688801" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T10:06:42.7881061+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/8/ROOT-1-130727775487940178" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T10:35:46.1418202+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/8/ROOT-1-130727793215154116" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T10:46:20.9861312+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/8/ROOT-1-130727799583188347" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T11:15:03.6796636+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/8/ROOT-1-130727816693406996" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T11:38:02.7185401+03:00" level="ERROR" thread="6" domain="/LM/W3SVC/8/ROOT-1-130727830639854687" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T11:56:56.6123952+03:00" level="ERROR" thread="13" domain="/LM/W3SVC/8/ROOT-1-130727841923780090" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T12:40:08.5816473+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/8/ROOT-1-130727867110710700" username="V-PC\V"><message>System.NullReferenceException: Ссылка на объект не указывает на экземпляр объекта.    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 44KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T12:40:40.0124451+03:00" level="ERROR" thread="15" domain="/LM/W3SVC/8/ROOT-1-130727867110710700" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T12:41:35.1075963+03:00" level="ERROR" thread="7" domain="/LM/W3SVC/8/ROOT-1-130727868825608787" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T12:46:08.2102169+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/8/ROOT-1-130727871528163364" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T12:49:20.7862316+03:00" level="ERROR" thread="7" domain="/LM/W3SVC/8/ROOT-1-130727873465024146" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T12:55:20.9168299+03:00" level="ERROR" thread="13" domain="/LM/W3SVC/8/ROOT-1-130727876891670139" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T13:04:23.3028526+03:00" level="ERROR" thread="7" domain="/LM/W3SVC/8/ROOT-1-130727882431947025" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T13:08:46.8429262+03:00" level="ERROR" thread="7" domain="/LM/W3SVC/8/ROOT-2-130727885066977740" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T13:11:58.0088603+03:00" level="ERROR" thread="7" domain="/LM/W3SVC/8/ROOT-1-130727887044830867" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T13:20:25.7239+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/8/ROOT-1-130727891998054175" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T13:21:32.004691+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/8/ROOT-1-130727892775648651" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T13:32:14.0564143+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/8/ROOT-1-130727899086729624" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T13:33:30.5377887+03:00" level="ERROR" thread="16" domain="/LM/W3SVC/8/ROOT-1-130727899086729624" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T13:35:53.1449454+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/8/ROOT-1-130727901105795108" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T13:49:54.8720894+03:00" level="ERROR" thread="6" domain="/LM/W3SVC/8/ROOT-1-130727909658314284" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T13:55:38.6417519+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/8/ROOT-1-130727913061218919" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T13:55:57.2908186+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/8/ROOT-1-130727913061218919" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T14:34:15.4802675+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/8/ROOT-1-130727936212073072" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T14:43:34.8812634+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/8/ROOT-1-130727941871846793" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T14:47:14.886847+03:00" level="ERROR" thread="9" domain="/LM/W3SVC/8/ROOT-1-130727944147006924" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T14:54:50.1058841+03:00" level="ERROR" thread="13" domain="/LM/W3SVC/8/ROOT-1-130727948636623716" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T15:02:06.1688254+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/8/ROOT-1-130727951814225464" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T15:16:25.5449789+03:00" level="ERROR" thread="17" domain="/LM/W3SVC/8/ROOT-2-130727961612085870" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T15:18:53.7644566+03:00" level="ERROR" thread="13" domain="/LM/W3SVC/8/ROOT-1-130727963045547859" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T15:21:10.440274+03:00" level="ERROR" thread="9" domain="/LM/W3SVC/8/ROOT-1-130727964463628969" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T15:37:51.3635236+03:00" level="ERROR" thread="23" domain="/LM/W3SVC/8/ROOT-2-130727973357387663" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T16:09:46.4840622+03:00" level="ERROR" thread="13" domain="/LM/W3SVC/8/ROOT-1-130727993606655855" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T16:12:09.339233+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/8/ROOT-1-130727995031977378" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T16:28:38.0737854+03:00" level="ERROR" thread="5" domain="/LM/W3SVC/8/ROOT-1-130728004831477878" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T16:33:39.3820193+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/8/ROOT-1-130728007787206936" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T16:37:07.0158952+03:00" level="ERROR" thread="13" domain="/LM/W3SVC/8/ROOT-1-130728010143101685" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T16:45:40.2892528+03:00" level="ERROR" thread="7" domain="/LM/W3SVC/8/ROOT-1-130728015150538094" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T16:51:25.4299937+03:00" level="ERROR" thread="13" domain="/LM/W3SVC/8/ROOT-1-130728018707211524" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T16:53:52.4814046+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/8/ROOT-2-130728020087210456" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-06T16:58:28.3551837+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/8/ROOT-1-130728022821346839" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 40KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-07T08:46:56.5906256+03:00" level="ERROR" thread="18" domain="/LM/W3SVC/8/ROOT-1-130728591425683918" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 41KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-07T08:54:53.9829309+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/8/ROOT-1-130728596705285894" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 41KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-07T08:55:57.8085815+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/8/ROOT-1-130728597436997746" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 41KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-07T09:01:08.1643329+03:00" level="ERROR" thread="14" domain="/LM/W3SVC/8/ROOT-1-130728600408767721" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 41KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-07T09:04:26.8976998+03:00" level="ERROR" thread="9" domain="/LM/W3SVC/8/ROOT-1-130728602479096137" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 41KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-07T09:12:17.5126174+03:00" level="ERROR" thread="14" domain="/LM/W3SVC/8/ROOT-1-130728606811843956" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\REP\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 41KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-08T10:42:11.9331999+03:00" level="ERROR" thread="7" domain="/LM/W3SVC/2/ROOT-1-130729525091228952" username="Кирюха-ПК\Кирюха"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\Кирюха\Desktop\Репа\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 41KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-09T14:23:05.3366422+03:00" level="ERROR" thread="5" domain="/LM/W3SVC/2/ROOT-1-130730520691997337" username="hpProBook4340s\SIVAS"><message>System.NullReferenceException: Ссылка на объект не указывает на экземпляр объекта.    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\SIVAS\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 45KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="hpProBook4340s" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-09T14:23:16.6695881+03:00" level="ERROR" thread="5" domain="/LM/W3SVC/2/ROOT-1-130730520691997337" username="hpProBook4340s\SIVAS"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\SIVAS\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 41KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="hpProBook4340s" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-09T14:23:47.3039988+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/2/ROOT-1-130730520691997337" username="hpProBook4340s\SIVAS"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\SIVAS\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 41KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="hpProBook4340s" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-09T14:29:33.0327078+03:00" level="ERROR" thread="6" domain="/LM/W3SVC/2/ROOT-1-130730525268160422" username="hpProBook4340s\SIVAS"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\SIVAS\Desktop\Rep\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 41KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="hpProBook4340s" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-14T14:08:14.3684516+03:00" level="ERROR" thread="7" domain="/LM/W3SVC/2/ROOT-1-130734832711011208" username="Кирюха-ПК\Кирюха"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\Кирюха\Desktop\Репазиторий\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 41KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-15T10:43:55.3074952+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/2/ROOT-1-130735574106080824" username="Кирюха-ПК\Кирюха"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\Кирюха\Desktop\Репазиторий\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 43KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-15T10:48:07.4069144+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/2/ROOT-1-130735576512108441" username="Кирюха-ПК\Кирюха"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\Кирюха\Desktop\Репазиторий\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 43KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-15T11:37:50.8795593+03:00" level="ERROR" thread="7" domain="/LM/W3SVC/2/ROOT-1-130735606468461847" username="Кирюха-ПК\Кирюха"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\Кирюха\Desktop\Репазиторий\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 43KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-15T11:38:23.5964306+03:00" level="ERROR" thread="6" domain="/LM/W3SVC/2/ROOT-1-130735606468461847" username="Кирюха-ПК\Кирюха"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\Кирюха\Desktop\Репазиторий\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 43KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-15T11:38:49.9449376+03:00" level="ERROR" thread="7" domain="/LM/W3SVC/2/ROOT-1-130735606468461847" username="Кирюха-ПК\Кирюха"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\Кирюха\Desktop\Репазиторий\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 43KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-15T11:39:17.5145145+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130735606468461847" username="Кирюха-ПК\Кирюха"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\Кирюха\Desktop\Репазиторий\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 43KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-15T11:40:23.366281+03:00" level="ERROR" thread="16" domain="/LM/W3SVC/2/ROOT-1-130735606468461847" username="Кирюха-ПК\Кирюха"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\Кирюха\Desktop\Репазиторий\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 43KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-15T11:40:39.3671962+03:00" level="ERROR" thread="16" domain="/LM/W3SVC/2/ROOT-1-130735606468461847" username="Кирюха-ПК\Кирюха"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\Кирюха\Desktop\Репазиторий\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 43KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-15T12:54:40.0171869+03:00" level="ERROR" thread="9" domain="/LM/W3SVC/2/ROOT-1-130735652025887582" username="Кирюха-ПК\Кирюха"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\Кирюха\Desktop\Репазиторий\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 43KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-15T12:54:55.370065+03:00" level="ERROR" thread="7" domain="/LM/W3SVC/2/ROOT-1-130735652025887582" username="Кирюха-ПК\Кирюха"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\Кирюха\Desktop\Репазиторий\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 43KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-15T13:41:27.2747528+03:00" level="ERROR" thread="9" domain="/LM/W3SVC/2/ROOT-1-130735680542588644" username="Кирюха-ПК\Кирюха"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\Кирюха\Desktop\Репазиторий\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 43KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-15T15:00:39.9695914+03:00" level="ERROR" thread="13" domain="/LM/W3SVC/2/ROOT-1-130735725095696936" username="Кирюха-ПК\Кирюха"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\Кирюха\Desktop\Репазиторий\trunk\KPIproduction\KPIWeb\KPIWeb\Account\UserLogin.aspx.cs:строка 43KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-21T10:41:01.6069669+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/2/ROOT-1-130740741999373642" identity="newuser1@mail.ru" username="Кирюха-ПК\Кирюха"><message>Пользователь newuser1@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-21T10:41:03.8470951+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/2/ROOT-1-130740741999373642" identity="newuser1@mail.ru" username="Кирюха-ПК\Кирюха"><message>Пользователь newuser1@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-21T10:41:06.3852402+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/2/ROOT-1-130740741999373642" identity="newuser1@mail.ru" username="Кирюха-ПК\Кирюха"><message>Пользователь newuser1@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-21T10:41:08.6203681+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/2/ROOT-1-130740741999373642" identity="newuser1@mail.ru" username="Кирюха-ПК\Кирюха"><message>Пользователь newuser1@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-21T10:41:10.8344947+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/2/ROOT-1-130740741999373642" identity="newuser1@mail.ru" username="Кирюха-ПК\Кирюха"><message>Пользователь newuser1@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-21T10:41:13.0556218+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/2/ROOT-1-130740741999373642" identity="newuser1@mail.ru" username="Кирюха-ПК\Кирюха"><message>Пользователь newuser1@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-21T10:41:15.2837492+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/2/ROOT-1-130740741999373642" identity="newuser1@mail.ru" username="Кирюха-ПК\Кирюха"><message>Пользователь newuser1@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-04-21T10:41:18.0319064+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/2/ROOT-1-130740741999373642" identity="newuser1@mail.ru" username="Кирюха-ПК\Кирюха"><message>Пользователь newuser1@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="Кирюха-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:56:48.1424082+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:56:50.2685298+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:56:52.3936514+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:56:54.5177729+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:56:56.6438945+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:56:58.769016+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:57:00.9571412+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:57:03.0902632+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:57:05.2423863+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:57:07.3685079+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:57:09.5136306+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:57:11.6417523+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:57:13.8008758+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:57:15.972+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:57:18.1241231+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:57:20.274246+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:57:22.4093682+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:57:24.53949+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:57:26.6616114+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:57:28.8097342+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:57:30.9328557+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:57:33.0599773+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:57:35.1960995+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:57:37.3912251+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:57:39.5703497+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:57:41.7364736+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:57:43.9195985+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:57:46.079722+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:57:48.2678472+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:57:50.4759735+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:57:52.6160959+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:57:54.7712192+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:57:57.0203478+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:57:59.1834715+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:58:01.394598+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:58:03.5957239+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/2/ROOT-1-130770357750245140" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 9 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:59:08.6314437+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:59:10.804568+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:59:12.9646916+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:59:15.1418161+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:59:17.2969394+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:59:19.4480624+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:59:21.6191866+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:59:23.7893107+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:59:25.9544345+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:59:28.1545604+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:59:30.3416855+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:59:32.4998089+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:59:34.633931+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:59:36.7880542+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:59:38.9481777+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:59:41.1083013+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:59:43.2694249+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:59:45.4435492+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:59:47.6116733+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:59:49.8057988+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:59:52.0019244+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:59:54.1790489+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:59:56.3881752+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T16:59:58.5973016+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:00:00.7754262+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:00:02.9265492+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:00:05.0506707+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:00:07.1797925+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:00:09.321915+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:00:11.4580372+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:00:13.5861589+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:00:15.7362819+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:00:17.8604034+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:00:19.9825248+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:00:22.1046461+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:00:24.2627696+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:00:26.5138983+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:00:28.693023+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:00:30.897149+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:00:33.0142701+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:00:35.1753937+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:00:37.3975208+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:00:39.5926464+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:00:41.7547701+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:00:43.9438953+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:00:46.1330205+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:00:48.3261459+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:00:50.5052706+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:00:52.6343923+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:00:54.7885156+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:00:56.9366384+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:00:59.0877615+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:01:01.2988879+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:01:03.4880131+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:01:05.6651377+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:01:07.8632634+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:01:10.0683895+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:01:12.2535145+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:01:14.4546404+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:01:16.6127638+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-25T17:01:18.7868882+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/2/ROOT-1-130770358863958840" identity="pvage@mail.ru" username="V-ПК\V"><message>Пользователь pvage@mail.ru сгенерировал ошибку 5 в отчете с ID = </message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-27T13:43:48.2205286+03:00" level="ERROR" thread="13" domain="/LM/W3SVC/2/ROOT-1-130771970063922801" username="V-ПК\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Projects.git\trunk\KPIWeb 3004FinalTest Vahe Version\KPIWeb\Account\UserLogin.aspx.cs:строка 43KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-27T14:32:15.5598189+03:00" level="ERROR" thread="28" domain="/LM/W3SVC/2/ROOT-1-130771997007643894" username="V-ПК\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Projects.git\trunk\KPIWeb 3004FinalTest Vahe Version\KPIWeb\Account\UserLogin.aspx.cs:строка 43KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-27T14:47:44.6429594+03:00" level="ERROR" thread="39" domain="/LM/W3SVC/2/ROOT-1-130771997007643894" username="V-ПК\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Projects.git\trunk\KPIWeb 3004FinalTest Vahe Version\KPIWeb\Account\UserLogin.aspx.cs:строка 43KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-27T14:50:18.5357616+03:00" level="ERROR" thread="33" domain="/LM/W3SVC/2/ROOT-1-130771997007643894" username="V-ПК\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Projects.git\trunk\KPIWeb 3004FinalTest Vahe Version\KPIWeb\Account\UserLogin.aspx.cs:строка 43KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-27T15:13:59.1560164+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/2/ROOT-1-130772023947544768" username="V-ПК\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Projects.git\trunk\KPIWeb 3004FinalTest Vahe Version\KPIWeb\Account\UserLogin.aspx.cs:строка 43KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-27T15:20:12.4903699+03:00" level="ERROR" thread="17" domain="/LM/W3SVC/2/ROOT-1-130772023947544768" username="V-ПК\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Projects.git\trunk\KPIWeb 3004FinalTest Vahe Version\KPIWeb\Account\UserLogin.aspx.cs:строка 43KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-27T15:20:58.6420097+03:00" level="ERROR" thread="22" domain="/LM/W3SVC/2/ROOT-1-130772023947544768" username="V-ПК\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Projects.git\trunk\KPIWeb 3004FinalTest Vahe Version\KPIWeb\Account\UserLogin.aspx.cs:строка 43KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-27T15:22:10.8681407+03:00" level="ERROR" thread="19" domain="/LM/W3SVC/2/ROOT-1-130772023947544768" username="V-ПК\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Projects.git\trunk\KPIWeb 3004FinalTest Vahe Version\KPIWeb\Account\UserLogin.aspx.cs:строка 43KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-27T15:24:57.4356679+03:00" level="ERROR" thread="23" domain="/LM/W3SVC/2/ROOT-1-130772023947544768" username="V-ПК\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Projects.git\trunk\KPIWeb 3004FinalTest Vahe Version\KPIWeb\Account\UserLogin.aspx.cs:строка 43KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-27T15:31:46.5890701+03:00" level="ERROR" thread="9" domain="/LM/W3SVC/2/ROOT-1-130772034693169383" username="V-ПК\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Projects.git\trunk\KPIWeb 3004FinalTest Vahe Version\KPIWeb\Account\UserLogin.aspx.cs:строка 43KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-27T15:39:00.0718639+03:00" level="ERROR" thread="36" domain="/LM/W3SVC/2/ROOT-1-130772034693169383" username="V-ПК\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Projects.git\trunk\KPIWeb 3004FinalTest Vahe Version\KPIWeb\Account\UserLogin.aspx.cs:строка 43KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-05-27T16:25:42.6381615+03:00" level="ERROR" thread="6" domain="/LM/W3SVC/2/ROOT-1-130772067265322403" username="V-ПК\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.UserLogin.Button1_Click(Object sender, EventArgs e) в c:\Users\V\Desktop\Projects.git\trunk\KPIWeb 3004FinalTest Vahe Version\KPIWeb\Account\UserLogin.aspx.cs:строка 43KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-ПК" /></properties></event>
